frundenbot/main.py

```diff
@@ -58,124 +99,110 @@ def __log_incomming_messages(bot, update):
+        context.bot.answerInlineQuery(update.inline_query.id, results)
+
+
+    CACHE_REFRESH_TIME = Summary('frunde_cache_refresh_seconds', 'Time spent refreshing cache')
+    @CACHE_REFRESH_TIME.time()
+    def refresh_cache(self, context: CallbackContext):
+        global cache
         try:
-            with open('/var/frunde/frunde_drinks.txt', 'w+') as file:
-                file.write('{}\n(Aktualisiert: {})'.format(mate_message, time.strftime('%d.%m.%Y um %H:%M')))
-                result = 'Neuer Matepegel:\n{}'.format(mate_message)
+            LOGGER.debug('Refresh cache')
+            r = requests.get('https://watchyour.freitagsrunde.org')
+            r.raise_for_status()
+            if 'Wir sind fuer dich da!' in r.text:
+                self.FRUNDE_OPEN.set(1)
+                cache = emojize(':white_check_mark: Die Freitagsrunde ist offen!', use_aliases=True)
+            else:
+                self.FRUNDE_OPEN.set(0)
+                cache = emojize(':red_circle: Leider haben wir gerade zu.', use_aliases=True)
         except Exception as e:
-            result = emojize('Uhm, das hat nicht geklappt. :confused:', use_aliases=True)
+            self.FRUNDE_OPEN.set(-1)
+            cache = emojize('Sorry, ich weiß es nicht! :confused:', use_aliases=True)
             LOGGER.error(e)
-        context.bot.sendMessage(chat_id=update.message.chat_id, text=result)
-        context.bot.sendMessage(chat_id=ADMIN, text='Neuer Matepegel von {} ({}):\n{}'.format(update.message.from_user.name, update.message.chat_id, result))
-    else:
-        context.bot.sendMessage(chat_id=update.message.chat_id, text=emojize(':poop: Nö :poop:', use_aliases=True))
-
-def main():
-    start_http_server(8000)
-    updater = Updater(token=TOKEN, use_context=True)
-    dispatcher = updater.dispatcher
-    queue = updater.job_queue
-
-    queue.run_repeating(refresh_cache, interval=60, first=0)
-
-    start_handler = CommandHandler('start', start)
-    inline_handler = InlineQueryHandler(inline)
-    open_handler = CommandHandler('open', is_open)
-    offen_handler = CommandHandler('offen', is_open)
-    drinks_handler = CommandHandler('drinks', get_drinks)
-    mate_handler = CommandHandler('mate', get_drinks)
-    whoami_handler = CommandHandler('whoami', whoami)
-    set_mate_handler = CommandHandler('set_mate', set_drinks)
-    message_handler = MessageHandler(Filters.text, is_open)
-
-    dispatcher.add_handler(start_handler)
-    dispatcher.add_handler(inline_handler)
-    dispatcher.add_handler(open_handler)
-    dispatcher.add_handler(drinks_handler)
-    dispatcher.add_handler(mate_handler)
-    dispatcher.add_handler(whoami_handler)
-    dispatcher.add_handler(set_mate_handler)
-    dispatcher.add_handler(offen_handler)
-    dispatcher.add_handler(message_handler)
-
-    updater.start_polling()
-
-if __name__ == '__main__':
-    main()
+
+
+@click.command()
+@click.option('--token', envvar='TOKEN', help='Telegram bot token.', required=True)
```

Comment:
IMO those envvar names are way to general, but since it is probably only used via docker I guess its fine...

Reply:
Yep, I was thinking about this as well. I prefixed them with `FRUNDE_` now.